helpers.ts

```diff
@@ -68,15 +68,22 @@ export function useSessionStorage(key, initialValue) {
   return [storedValue, setValue];
 }
 
-export function useEmailCloaker(initialValue) {
-  const email = addrs.parseOneAddress(initialValue);
-  return [
-    email?.local.charAt(0),
-    email?.local.charAt(email?.local.length - 1),
-    `@${email?.domain}`,
-  ];
+export function useEmailCloaker(initialValue: string): string[] | [] {
+  const email = parseOneAddress(initialValue);
+  if (email && "local" in email && "domain" in email) {
+    return [
+      email.local.charAt(0),
+      email.local.charAt(email.local.length - 1),
+      `@${email.domain}`,
+    ];
+  }
+  return [];
```

Comment:
Ah, so, it looks like the only implementation of `useEmailCloaker()` is when we're adding members via `addMember()`.

And this... doesn't actually make a lot of sense because we immediately convert it to a string in that same addMember function.
```
const maskedEmail = useEmailCloaker(member.email);
const maskedEmailString = `${maskedEmail[0]}...${maskedEmail[1]}${maskedEmail[2]}`;
```

So okay, I made some tweaks to both `useEmailCloaker()` and `addMember()` functions. It now just returns the "masked email string", saving us this roundabout step. But this also necessitated catching errors via a try catch function.